scalableminds/webknossos
# Validate dataset name before upload

### URL of deployed dev instance (used for testing):
- https://validatedatasetname.webknossos.xyz

### Steps to test:
- in dataset upload form, type dataset name that is already in use, error should appear after short delay
- fix the name, error should go away

### Issues:
- fixes #1901 
------
